storage.js

```diff
@@ -0,0 +1,27 @@
+const contactForm = document.querySelector('#contact-form');
+
+function storeData(nameData, emailData, messageData) {
+  const formData = {
+    nameData,
+    emailData,
+    messageData,
+  };
+
+  localStorage.setItem('formData', JSON.stringify(formData));
+  // console.log(JSON.stringify(formData));
+}
+contactForm.addEventListener('keyup', (e) => {
+  e.preventDefault();
+  const name = document.getElementById('first-name').value;
+  const email = document.getElementById('email').value;
+  const message = document.getElementById('message').value;
+  storeData(name, email, message);
+});
+
+const getData = JSON.parse(localStorage.getItem('formData'));
+
+window.onload = () => {
+  document.getElementById('first-name').value = getData.nameData;
+  document.getElementById('email').value = getData.emailData;
+  document.getElementById('message').value = getData.messageData;
+};
```

Comment:
- [ ] The features you've implemented are technically working although Please create a check to make sure there is data in the `localStorage` before trying to access `getData.nameData`. currently, you have errors in the console when there is no localStorage because `getData` will be `null` when there is no `localStorage`. thus you can't access `namedata/emailData/messageData`. Please implement this change and your project should be complete.
![image](https://user-images.githubusercontent.com/18730106/151403544-ecf8ccfd-4de8-4b26-b4ad-595bb1681f68.png)